consensus/safety-rules/src/safety_rules.rs

```diff
@@ -233,16 +245,11 @@ impl TSafetyRules for SafetyRules {
                 last_voted_round: self.persistent_storage.last_voted_round()?,
             });
         }
+        proposed_block.validate_signatures(self.verifier()?)?;
```

Comment:
we could also add to this stack a commit that renames validate_signatures to validate_signature, I don't know why this was ever named to be plural.